db/migrate/20220515130414_drop_unnecessary_timestamps.rb

```diff
@@ -0,0 +1,22 @@
+class DropUnnecessaryTimestamps < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
This would probably be easier to read if instead of `change { reversible { up {} down {} } }` we just went with `def up` and `def down`

Reply:
Docs [0] says that you can use only `change` method, it should automatically know how to handle migration / rollback for the `remove_column` method [1]

[0] https://guides.rubyonrails.org/active_record_migrations.html#using-the-change-method
[1] https://api.rubyonrails.org/v6.1/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-remove_column

Reply:
That's true, but it doesn't know what data to fill into the column on rollback, or does it?

Reply:
See https://github.com/theforeman/foreman/blob/develop/db/migrate/20200525073201_drop_puppet_class_id_from_look_up_key.rb

Reply:
> That's true, but it doesn't know what data to fill into the column on rollback, or does it?

Yeah it does not, but we don't use the columns so technically it doesn't matter in this case.